docs/ctr-remote.md

```diff
@@ -2,7 +2,7 @@
 
 This doc describes example usages of `ctr-remote image optimize` command for converting images into eStargz.
 
-`ctr-remote images optimize` command (call `ctr-remote` in this doc) enables users to convert an image into eStargz.
+`ctr-remote image optimize` command (call `ctr-remote` in this doc) enables users to convert an image into eStargz.
```

Comment:
Well, this is not a typo. Both of them are valid subcommands. See https://github.com/containerd/containerd/blob/ad447624cd5aefb387ad67482b3aecdf2e9f2acd/cmd/ctr/commands/images/images.go#L38